### PR TITLE
Add _tkinter troubleshooting

### PR DESCRIPTION
**Pull request by gamer4life1**
Using my custom tap ^-^

### CODE REVIEW

**Comment by Alvarito050506** [comment]
Reviewing!

**Comment by gamer4life1** [comment]
Thanks

File: README.md [hunk 1]
```diff
@@ -50,6 +50,25 @@ mcpim ./example.py
 ```
 This will produce an `example.mcpi` mod file.
 
+## Troubleshooting
+If you get this error: module not found: _tkinter or something of the sort, you can use homebrew and use a custom tap made by @gamer4life1.
```
Comment (Alvarito050506) [inline README.md:54]: I would change it to something like:
```
If you get the `module not found: _tkinter` error, or something of the sort, you can use Homebrew to install a custom tap made by @gamer4life1.
```

File: README.md [hunk 2]
```diff
@@ -50,6 +50,25 @@ mcpim ./example.py
...
+```shell
+# First, install brew (if you don't have it already (make sure to install in /home/linuxbrew/.linuxbrew
```
Comment (Alvarito050506) [inline README.md:56]: Here you have a few typos: unmatching `()`

**Review by Alvarito050506 — CHANGES_REQUESTED** [review]
Hello! Thanks for sending this PR! I suggested you a few changes, mostly related to typos and styling.

File: README.md (hunk 2 above)
Comment (gamer4life1) [reply README.md:56]: Ah oops

**Review by Alvarito050506 — APPROVED** [review]
Awesome! Merging...

**Pull request merged**